### DISCUSSION TITLE
@getgauge\cli\bin\gauge"' is not recognized as an internal or external command, operable program or batch file

### DISCUSSION THREAD

**Discussion by pattabhi007** [getgauge/gauge#2131] [General]
Hi Team,

Getting below error while running Gauge spec file. Could you please help.

@getgauge\cli\bin\gauge"' is not recognized as an internal or external command, operable program or batch file. 

Below is my version details.
"@getgauge/cli": "^1.4.1",
"taiko": "^1.2.7",

**Comment by sriv** [comment]
`@getgauge\cli\bin\gauge` should be `@getgauge\cli\bin\gauge.exe` on windows. Can you share when you get this error?

**Reply by pattabhi007** [reply]
Hi @sriv 

I am trying to run a feature file with below command. It used to work, but not sure what happened, it is not working any more. 

gauge run specs/ --tags=TCMTBYOC --env=uat

Below is the complete error screenshot.

![image](https://user-images.githubusercontent.com/28644123/136884373-dac17150-452e-46a7-9f14-48809d5f8426.png)


If give "gauge -v" command, a command prompt pops up, and disappears quickly as shown in below.
![image](https://user-images.githubusercontent.com/28644123/136883965-a8820634-a7f9-4da5-a747-d2add6a73759.png)

**Reply by sriv** [reply]
can you check if `gauge.exe` is available in `node_modules\@getgauge\cli\bin`?

**Comment by pattabhi007** [comment]
Hi mate, gauge.exe exists in node_modules\@getgauge\cli\bin. But not sure why it fails. 

I wiped out my local repo, and reconfigured. It worked after that. Thanks for assisting.
